Slim/Networking/Async/Socket/HTTPsocks.pm

```diff
@@ -0,0 +1,19 @@
+package Slim::Networking::Async::Socket::HTTPsocks;
+
+use strict;
+
+use base qw(IO::Socket::Socks Net::HTTP::Methods Slim::Networking::Async::Socket);
```

Comment:
Is `IO::Socket::Socks` pure Perl? Could you please include it in the PR?

Reply:
Yes it is pure Perl, no headache with compilation. I'll add it